demos/object_detection_demo_faster_rcnn/detectionoutput.h

```diff
@@ -107,7 +107,7 @@ class DetectionOutputPostProcessor {
                     {Precision::I32, indices_size, {indices_size, {0, 1, 2}}});
             _indices->allocate();
 
-            SizeVector detections_size{static_cast<size_t>(_num * _num_classes)};
+            SizeVector detections_size{static_cast<size_t>(_num) * static_cast<size_t>(_num_classes)};
```

Comment:
It looks `_num` is never used as `int`. Except for `_num = static_cast<int>(conf_size);` but `conf_size` is `uint32_t`. `_num` should be defined as `size_t`.

Reply:
Good catch. `_num` is actually entirely redundant, so I just removed it.